_targets/build.common

```diff
@@ -60,16 +65,16 @@ b_mkscript_user() {
 
 					PROGS+=("$name")
 					psz=$(wc -c < "${PREFIX_PROG_STRIPPED}$name")
-					printf "alias %s 0x%x 0x%x\n" "$name" "$poffs" "$psz"
+					printf "alias%s %s 0x%x 0x%x\n" "$RELATIVE_ALIAS" "$name" "$poffs" "$psz"
```

Comment:
Simplification possibility - you can use -r always, as default base is zero. I'm just pointing that out, I'm not sure it's a good idea. @anglov ?

Reply:
I was also thinking about backward compatibility. Any project that doesn't upgrade plo on programming will fail due to lack of relative alias